### PR TITLE
feat(api): setup prisma mock for tests

### LINKED ISSUE #873: chore: fix commented out tests on api package
https://github.com/gitcoinco/grants-round/pull/856#issuecomment-1358716914

### PR DESCRIPTION
**Pull request by chibie**
##### Description

This PR mocks prisma query and fixes commented out API handler tests

##### Refers/Fixes

closes #873

### CODE REVIEW

**Comment by melissa-neira** [comment]
defer to engineers 🕺

**Review by thelostone-mc — APPROVED** [review]
LGTM once conflicts are resolved

**Comment by nfrgosselin** [comment]
Defer to engineering review 🕺

**Pull request merged**
